tslint.json

```diff
@@ -5,5 +5,8 @@
     "tslint-eslint-rules",
     "tslint-config-prettier"
   ],
-  "rules": { "interface-name": [true, "never-prefix"] }
+  "rules": {
+    "interface-name": [true, "never-prefix"],
+    "no-var-requires": false
```

Comment:
Why do we need to change this rule?

Reply:
tslint wouldn't let me commit my changes because of this line of code right here
`const WebFontLoader = require('webfontloader')` in the mainsiteGlobalStyles.tsx file

@zeehang suggested to add this change

we could change that to this if you don't want to change any of the rules:
`import WebFontLoader from 'webfontloader`

and remove this line `const WebFontLoader = require('webfontloader')`

Reply:
let's do that!